cmds/core/pwd/pwd_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2012-2023 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestPWD(t *testing.T) {
+	dir := t.TempDir()
+	err := os.Chdir(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rs, err := filepath.EvalSymlinks(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	r, err := pwd(true)
```

Comment:
I wonder: it is nice to have a test now. Maybe on line 19 it's worth adding a test for the non-symlink case? It seems it would be simple to add, maybe it is not needed however?

Reply:
from coverage it will not help, but I it worth to add.